adapter/src/protocol.ts

```diff
@@ -111,6 +111,8 @@ export class ProtocolServer extends ee.EventEmitter implements VSCodeDebugAdapte
 				this._pendingRequests.delete(response.request_seq);
 				clb(response);
 			}
+		} else if (msg.type === 'event') {
+			this._emitEvent(<DebugProtocol.Event>msg)
```

Comment:
Not sure about this, it seems like the intent is not to emit DAP events but just errors and the like.

Reply:
The emitter is typed as emitting `DebugProtocolMessage`'s, but that's just an empty interface 🤷  I think it'd make sense to emit them here

Reply:
But it's weird that it's also emitting non-DAP errors from the same emitter. But also that events currently aren't going anywhere? Do you have some more context on how this works at all currently @ashgti?

Reply:
I wasn't sure if there was a way to get DAP events if you used the adapter programmatically, but when I added the types here I noticed that events weren't being handled.

I can revert this part, it just seemed like a missing piece of the adapter.

Reply:
I think this method is only handling the data stream from the client, so this would only be hit if the client sends an event. I don't think that happens for any event types, or does it?

Reply:
It doesn't happen for any event types, removed.